torch/csrc/api/include/torch/nn/options/embedding.h

```diff
@@ -21,7 +21,11 @@ struct TORCH_API EmbeddingOptions {
   TORCH_ARG(int64_t, num_embeddings);
   /// The size of each embedding vector.
   TORCH_ARG(int64_t, embedding_dim);
-  /// If given, pads the output with the embedding vector at `padding_idx` (initialized to zeros) whenever it encounters the index.
+  /// If specified, the entries at `padding_idx` do not contribute to the
+  /// gradient; therefore, the embedding vector at `padding_idx` is not updated
+  /// during training, i.e. it remains as a fixed “pad”. For a newly constructed
```

Comment:
Hmm, can you please explain what is semantic meaning behind unicode double quotes? Shouldn't this be just in single quotes, to indicate codeblock?
```suggestion
  /// during training, i.e. it remains as a fixed ``pad``. For a newly constructed
```